notebooks/pytorch_sparse_quantized_transfer_learning.ipynb

```diff
@@ -0,0 +1,467 @@
+{
```

Comment:
<p>Let's link out to the docs for DeepSparse hardware compatibility when mentioning the VNNI instructions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/neuralmagic/sparseml/pull/81/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>